INSTALL.rst

```diff
@@ -53,6 +53,12 @@ and then install from the local repository via::
     cd pypesto
     pip3 install .
 
+Install and run pyPESTO container
+---------------------------------
+
+.. include:: containers.rst
```

Comment:
The heading levels don't match the surrounding document, see https://pypesto--1539.org.readthedocs.build/en/1539/install.html#

Please fix or include as a separate page.